ftplugin/latex-suite/bibtools.py

```diff
@@ -27,8 +33,12 @@ def __init__(self, txt, macros={}):
         """
 
         if macros:
-            for k, v in macros.iteritems():
-                txt = txt.replace(k, '{' + v + '}')
+            try:
+                for k, v in macros.iteritems():
+                    txt = txt.replace(k, '{' + v + '}')
+            except AttributeError:
+                for k, v in macros.items():
+                    txt = txt.replace(k, '{' + v + '}')
```

Comment:
I think I just preserved the code for Python 2 that was already there. If you think it should be changed, it can be changed.